app/assets/stylesheets/components/_layout.scss

```diff
@@ -75,3 +75,32 @@
     }
   }
 }
+
+.back-to-top {
+  display: none;
+  float: right;
+  margin-top: 1.5rem;
+  position: sticky;
+  position: -webkit-sticky;
+  top: 1rem;
+
+  a {
+    @include padding(0.75rem 1.5rem);
+    background-color: $color-gold;
+    border-radius: $border-radius;
+    color: $color-base;
+    display: inline-block;
+    font-size: $small-font-size;
+    font-weight: $font-bold;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+  }
+
+  @include media($nav-width) {
+    display: block;
+  }
```

Comment:
This applies to the parent selector (`.back-to-top`), yeah? Recommend moving up to line 86-ish so its nearer the styling for that element.